### DISCUSSION TITLE
I NEED HELP PLEASE !!!

### DISCUSSION THREAD

**Discussion by mikewam** [OnionUI/Onion#777] [Support] [answered]
****I need help, I was upgrading from onion 4.2 beta to MM+ and I had a problem. my** system does not leave the miyoo boot screen. I already tried to remove the SDcard to see if it connects to the stock and the infinite boot screen continues. I also tried to reinstall the system and the boot screen problem continues. If anyone can help me please, I'm afraid I have bricked the device.**

**Comment by Aemiii91** [comment]
Did you charge the device recently?

**Reply by mikewam** [reply]
yes the device was charged, I did the tutorial procedure through a channel on YouTube, but in the part that the update rocket was supposed to appear, nothing appeared, as it took a long time I turned off the system and turned it on again and now every time I turn on the device stays on the device's boot screen and nothing happens 😰

( I'm sorry if I have any English mistakes I'm using google translator, I'm Brazilian)

**Reply by mikewam** [reply]
![IMG20230413200645.jpg](https://user-images.githubusercontent.com/130701098/231873145-b5b87da1-e607-485f-ba1b-b4a27d2596b3.jpg)

**Reply by Aemiii91** [reply]
OK, did you retry the firmware update?

**Reply by mikewam** [reply]
I tried again to do the process, and also tried to reinstall the factory system to see if it would solve the infinite boot problem continues.
Is it possible to do some hard reset or force this system to start somehow?
i am using a quality sdcard.

**Reply by mikewam** [reply]
**my device was in the Onion V4.2.0 version and I tried to update to the V4.2.0.1 version. Can you send me this previous beta version to see if I can start the console with these files from the previous beta?**

**Reply by Aemiii91** [reply]
That wouldn't make a difference. You need to retry the *firmware* update (`miyoo354_fw.img`, version 20230326)

**Reply by mikewam** [reply]
I will try to do it again, with another Sdcard the same procedure.  and tomorrow I will send you a feedback, thank you very much for your attention friend.  Few are willing to help.

**Comment by mikewam** [comment, marked as answer]
I managed to solve the problem. if possible friend can share this to help other people who had the same problem as mine will be a great pleasure. thank you for everything !

https://www.reddit.com/r/MiyooMini/comments/12j7tjf/comment/jhkd0e6/?utm_source=share&utm_medium=web2x&context=3
